jupyter_ydoc/ydoc.py

```diff
@@ -159,7 +329,19 @@ def create_ycell(self, value: Dict[str, Any]) -> None:
 
         return Y.YMap(cell)
 
-    def set_ycell(self, index: int, ycell: Y.YMap, txn=None):
+    def set_ycell(self, index: int, ycell: Y.YMap, txn=None) -> None:
```

Comment:
Worth updating the type annotation with what you wrote in the docstring?

Reply:
Do you mean like this: `def set_ycell(self, index: int, ycell: Y.YMap, txn: Optional[Y.Transaction]=None) -> None:` ?

Reply:
Yes, but it's `Y.YTransaction`.